SegmentEditorSurfaceCut/SegmentEditorSurfaceCutLib/SegmentEditorEffect.py

```diff
@@ -175,6 +192,17 @@ def onOperationSelectionChanged(self, operationName, toggle):
       return
     self.scriptedEffect.setParameter("Operation", operationName)
 
+  def onSmoothModelCheckboxStateChanged(self, newState):
+    logging.info('onSmoothModelCheckboxStateChanged, new state is: %s'%(str(newState)))

```

Comment:
no need to log this, but if you really want to then change it to debug level (info message is displayed in the Python console, which would be too much noise)

Reply:
Thanks, that was a debugging message, most of which I had put in as logging.debug, and I just missed this one when I was removing those.  I'll take this one out also

Reply:
Instead of adding temporary log messages, debugging is much easier and faster if you connect a debugger to Slicer (Visual Studio Code, PyCharm, VIsual Studio, and Eclipse are all supported). See more information here: https://www.slicer.org/wiki/Documentation/Nightly/Extensions/DebuggingTools